src/atomate2/forcefields/jobs.py

```diff
@@ -0,0 +1,81 @@
+"""Job to relax a structure using a force field (aka an interatomic potential)."""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass, field
+
+from jobflow import Maker, job
+from pymatgen.core.structure import Structure
+
+from atomate2.forcefields.schemas import FFStructureRelaxDocument
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class CHGNetRelaxMaker(Maker):
+    """
+    CHGNet relax maker.
+
+    Parameters
+    ----------
+    name: str
+        The job name.
+    relax_cell : bool
+        Whether to allow the cell shape/volume to change during relaxation.
+    relax_kwargs : dict
+        Keyword arguments that will get passed to :obj:`StructOptimizer.relax`.
+    optimizer_kwargs : dict
+        Keyword arguments that will get passed to :obj:`StructOptimizer()`.
+    keep_info : list
+        Which information from the relaxation trajectory to save using
+        the :obj:`.FFStructureRelaxDocument.from_chgnet_result`.
+
+    """
+
+    name: str = "CHGNet relax"
+    relax_cell: bool = False
+    relax_kwargs: dict = field(default_factory=dict)
+    optimizer_kwargs: dict = field(default_factory=dict)
+    keep_info: list = field(
```

Comment:
Remove this and put the default value on the task document. Add a new field `task_document_kwargs` that will get passed to the task document function. If people want to override this field then they can do something like:

```python
maker = CHGNetRelaxMaker(task_document_kwargs={"keep_info": ["energies"]})
```

This way is better since if we add more kwargs to the task document later on then we don't need to add new attributes to the Maker class.

Note, when you add the default to the task document function defaults, specify them as a tuple rather than a list.